### PR TITLE
Updated Khronos group logos to the last available version

### PR DESCRIPTION
**Pull request by jaibeer72**
### Purpose of the PR
* Updating the logos for Khronos group 
* Closing the issue number #2317 

### Technical Risk
* I do not see any technical risk or performance risk also 
* if the removal of the old png caused something I am sorry about that I searched the structure my best did not find anything.

### Comments to the reviewer 
* Still new to all of this I am using the procedure of work to kinda be inline here 
* Also I had to change the width and size of the logos to match the menu if required we can use CSS to make it more Aspect ratio based but I decided to stay consistent with initial design as I am still finding my way through the repository 
* I have usually learnt to create open branch commits to make sure if a test is failing to figure if it's my fault or not. that is why that exits.

### CODE REVIEW

**Comment by kenrussell** [comment]
Thanks for this contribution. Could you please confirm when you've signed the CLA?

Do these web pages all look good with the logo being that much bigger? If not, what about manually resizing one of the larger copies of the new logo to the original dimensions in these files, and checking that in instead?

**Comment by kenrussell** [comment]
BTW, the lint errors aren't your fault - if you rebase on top of master they should be gone.

**Comment by jaibeer72** [comment]
> Thanks for this contribution. Could you please confirm when you've signed the CLA?
> 
> Do these web pages all look good with the logo being that much bigger? If not, what about manually resizing one of the larger copies of the new logo to the original dimensions in these files, and checking that in instead?

Yeah, I already signed the CLA. 
about the logo size in lower resolutions, it seemed to be getting squashed if need be I will work on the size more. I also did check the Khronos brand guidelines to be sure all the HTML files are still abiding them.

**Comment by jaibeer72** [comment]
> BTW, the lint errors aren't your fault - if you rebase on top of the master they should be gone.

should I rebase the branch on top of the master ?  or will just merge it will fix it. (usually failed builds scare me 😏 ) 

PS:  again if you need me to make a PR interation here I am more than happy to do it. also if you can suggest some reading material to help better with overall code bad I would love to contribute more.

**Comment by jaibeer72** [comment]
@kenrussell you were right it looks better with a smaller image. I fixed that and hopefully, it's good now.

File: extensions/extension.xsl [hunk 1]
```diff
@@ -195,7 +195,7 @@
     <a href="http://webgl.org/"><img alt="WebGL" height="100" src="{concat($basepath,'/resources/WebGL-Logo.png')}" width="240" /></a>
   </div>
   <div class="right">
-    <a href="http://khronos.org/"><img alt="Khronos" height="60" src="{concat($basepath,'/resources/KhronosGroup-3D.png')}" width="220" /></a>
+    <a href="http://khronos.org/"><img alt="Khronos" height="80" src="{concat($basepath,'/resources/Khronos_100px_June18.png')}" width="336" /></a>
```
[the remark this answers is not in the document]
Comment (jaibeer72) [reply extensions/extension.xsl]: Yeah so I checked for 252x60 but it seemed a little off with regards to the webGL logo ill send you a few artefacts regarding it later

File: specs/1.0.0/index.html [hunk 2]
```diff
@@ -22,7 +22,7 @@
         <div class="right">
             <p>
                 <a href="http://khronos.org/">
-                    <img alt=Khronos height=60 src="../../resources/KhronosGroup-3D.png" width=220>
+                    <img alt=Khronos width=336 height=80 src="../../resources/Khronos_100px_June18.png"/>
```
[the remark this answers is not in the document]
Comment (jaibeer72) [reply specs/1.0.0/index.html]: I did find as many as I could for Khronos and webGL that did not seem to be risky to mess with but I think we should also work on a Coding Style docs and naming conventions Wiki as well so contributions can be more consistent. What do you think? If there is one and I missed it sorry 🙈

[the remark this answers is not in the document]
Comment (kainino0x) [reply specs/1.0.0/index.html]: It's not so much a matter of following a particular coding style. We just want to stay consistent. So, for example, if someone wanted to more easily change these all again later, they could just paste the same replacement line for every one without any extra changes.

IIRC our coding style is basically defined by "match the style of surrounding code", plus whatever style comments reviewers have.

[the remark this answers is not in the document]
Comment (kainino0x) [reply specs/1.0.0/index.html]: To the extent we have a style guide, it is here:
https://github.com/KhronosGroup/WebGL/blob/master/sdk/tests/test-guidelines.md

But it is about the code style of the tests themselves, not surrounding stuff like the HTML.

[the remark this answers is not in the document]
Comment (jaibeer72) [reply specs/1.0.0/index.html]: Thanks, ill make sure to look into it and

**Comment by jaibeer72** [comment]
I am hoping everything is good now :D

**Comment by jdarpinian** [comment]
Thanks. Now that the image is the same size on every page, can you resize the image file itself to match? That way the browser won't have to scale it.

**Comment by jaibeer72** [comment]
@jdarpinian Done :D 
I think i made this PR more Complicated than it needed to be and made you all put much more effort than it should have been but again if anything else I am all for it. 
Thanks

**Comment by kainino0x** [comment]
I just checked out how the images look in a browser; looks great! (Matches better with the size of the WebGL logo now.) Just pushed some tiny changes to normalize a bit more of the formatting (like indenting a line that was already wrong before your patch). Merging now. Thanks!

**Pull request merged**